.github/workflows/maven.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         # test against latest update of each major Java version, as well as specific updates of LTS versions:
-        java: [ 8, 11, 13 ]
+        java: [ 8, 11, 13, 15 ]
```

Comment:
According to the comment above I would have expected 15 to replace 13 rather than getting appended? But this works too of course.

Reply:
Don't really have a fully thought-out strategy here, but went with both 13 and 15 as they are MTS releases on some distros (e.g. Azul https://www.azul.com/products/azul-support-roadmap/). Once 17 gets released, I would imagine we would remove 13 and 15.